packages/orchestration/src/utils/denomHash.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
+import { sha256 } from '@noble/hashes/sha256';
+// import { toHex } from '@cosmjs/encoding';
+
+// ack: https://stackoverflow.com/a/40031979/7963
+function toHex(buffer) {
+  // buffer is an ArrayBuffer
+  return [...new Uint8Array(buffer)]
+    .map(x => x.toString(16).padStart(2, '0'))
+    .join('');
```

Comment:
Reasonably concise. An intermediate allocation might be avoidable with `Array.prototype.map.call(new Uint8Array(buffer), x => x.toString(16).padStart(2, '0')`.

Reply:
turns out `@noble/hashes/utils` has a `bytesToHex`, so I'm going with that.